.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+cache: yarn
+script:
+- yarn test
+- rvm install 2.3.1
```

Comment:
There is a `pretest` script that runs the linter, but yeah we should probably add flow to that as well.